pkg/crypto/webkms/remotecrypto.go

```diff
@@ -335,7 +338,18 @@ func (r *RemoteCrypto) Verify(signature, msg []byte, keyURL interface{}) error {
 
 // ComputeMAC remotely computes message authentication code (MAC) for code data with key at keyURL.
 // using a matching MAC primitive in kh key handle.
-func (r *RemoteCrypto) ComputeMAC(data []byte, keyURL interface{}) ([]byte, error) {
+func (r *RemoteCrypto) ComputeMAC(data []byte, keyURL interface{}) ([]byte, error) { //nolint: gocyclo
+	if r.opts.ComputeMACCache != nil {
+		v, err := r.opts.ComputeMACCache.Get(string(data))
```

Comment:
The cache key should be both data and keyuRL

Reply:
done